witnet/prices/a/AdaUsdPrice.js

```diff
@@ -63,8 +63,8 @@ const request = new Witnet.Request()
   .setAggregator(aggregator) // Set the aggregator function
   .setTally(tally) // Set the tally function
   .setQuorum(10, 51) // Set witness count and minimum consensus percentage
-  .setFees(10 ** 6, 10 ** 6) // Set economic incentives
-  .setCollateral(5 * 10 ** 9) // Require 5 wits as collateral
+  .setFees(15 * 10 ** 7, 10 ** 7) // Witnessing fee: 0.1 wit; Commit/Reveal fee: 0.01 wit;
+  .setCollateral(15 * 10 ** 8) // Require 5 wits as collateral
```

Comment:
This is 1.5 wits of collateral, and the comments are outdated.

Perhaps instead of hardcoding the collateral and the fees we could use a constant defined somewhere else, and simply import that constant in the price feeds? This way it will be easier to change in the future.

Reply:
+1 to that

Reply:
Done